### ISSUE TITLE
Total vaccination numbers are all 0

### ISSUE THREAD

**Issue by mawalu** [marlon360/rki-covid-api#112]
The api currently returns `0` for all of the accumulated vaccination statistics. I checked the xlsx file and everything there seems fine. There exists a new row `Bund (Einsatzkräfte Bundeswehr, Bundespolizei)` that I haven't seen before, maybe that is causing some problems.

```json
{
  "administeredVaccinations": 0,
  "vaccinated": 0,
  "vaccination": {
    "biontech": 0,
    "moderna": 0,
    "astraZeneca": 0
  },
  "delta": 0,
  "quote": 0,
  "secondVaccination": {
    "vaccinated": 0,
    "vaccination": {
      "biontech": 0,
      "moderna": 0
    },
    "delta": 0,
    "quote": 0
  },
  "indication": {
    "age": 0,
    "job": 0,
    "medical": 0,
    "nursingHome": 0,
    "secondVaccination": {
      "age": 0,
      "job": 0,
      "medical": 0,
      "nursingHome": 0
    }
  }
}
```

**Comment by marlon360** [comment]
Thanks @mawalu
I will fix that!

**Comment by marlon360** [comment]
Fixed with 975e2d91e5d2b63dc4e87dd317537fec25346966
